courscala/src/main/scala/org/coursera/common/stringkey/StringKeyFormat.scala

```diff
@@ -235,6 +236,12 @@ sealed trait CommonStringKeyFormats extends DefaultTupleFormats {
   implicit val instantFormat: StringKeyFormat[Instant] =
     StringKeyFormat.delegateFormat[Instant, Long](millis => Some(new Instant(millis)), _.getMillis)
 
+  implicit def seqFormat[T](implicit format: StringKeyFormat[T]):
```

Comment:
Add the deprecation warnings or Scaladoc that warns about not using it because of its slight incorrectness?